docs/REST API/rest-api.md

```diff
@@ -9032,6 +9032,7 @@ Returns information about the specified software. By default, `versions` are sor
       "app_store_id": 1091189122,
       "latest_version": "2.04",
       "icon_url": "https://is1-ssl.mzstatic.com/image/thumb/Purple211/v4/f1/65/1e/a4844ccd-486d-455f-bb31-67336fe46b14/AppIcon-1x_U007emarketing-0-7-0-85-220-0.png/512x512bb.jpg",
+      "self_service": true,
```

Comment:
#### Dev note:

In `GET /api/v1/fleet/software/titles/:id` add `self_service` flag to `app_store_app` object when VPP app is added as self-service app.

Reply:
Is self-service a property of the app's association with a team, or the application itself?

Since we specify `self_service` on the team spec, I would assume the property belongs to the association. That would mean that a single title can be both self service and not self service based on the team

Reply:
@noahtalerman I see marko is OOO, can you take a look at this?

Reply:
Hey @dantecatalfamo, self-service is a property of the app not the team. An app can either be self service or not self service (not both).

Where do we specify `self_service` in the team spec?

Reply:
On the batch endpoint, we're associating an app with a team and passing the `self_service` argument https://github.com/fleetdm/fleet/pull/19687/files#diff-831a63ebb3cab9b4e6b82d803d9ffcdc3722b12d486dcd7b13e9576643bb50b0R3051

Reply:
There's nothing stopping a user from specifying `self_service: true` on one team and `self_service: false` on another in the gitops yaml

Reply:
And on #19882 the suggested migration adds it to the app team join table

Reply:
My understanding is that self_servie is a property of the app as it is made available to the team, not the title itself

Reply:
> There's nothing stopping a user from specifying self_service: true on one team and self_service: false on another in the gitops yaml

> that self_servie is a property of the app as it is made available to the team, not the title itself

I think this is right.

@dantecatalfamo If I'm understanding correctly, this means that you could have Google Chrome in self-service for Team A but not in self-service for Team B.

This is what we want.

I think that's ok.

In the Fleet DB this will actually be two Google Chrome apps (one for Team A and one for Team B) associated with the same software title.

Reply:
@noahtalerman Ah! Okay that helps a lot, thank you! I misunderstood the association with `titles`